### DISCUSSION TITLE
Is k3s planning to move upwards as a CNCF Incubated project?

### DISCUSSION THREAD

**Discussion by mauromorales** [k3s-io/k3s#11479] [General]
Hi everyone, I'm just trying to understand CNCF projects better, and I have a hard time understanding why a project like k3s, which has such good adoption and has been battle tested, is still in the Sandbox. Is this because you, the maintainers, want to keep it there, or have you tried to move up but didn't get approved?

**Comment by brandond** [comment]
Waiting on TOC. Apparently they're shorthanded? It's been sitting since April.
* https://github.com/cncf/toc/pull/1307

**Reply by mauromorales** [reply]
@brandond thanks for the prompt reply and very cool 🙌 ... It didn't occur to me to look in the open PRs 🤦 but I'm thrilled to hear this is the case. Let's hope response comes soon 🤞
